pkg/cmd/migrate.go

```diff
@@ -0,0 +1,95 @@
+package cmd
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	_ "github.com/jackc/pgx/v4/stdlib"
+	"github.com/openfga/openfga/assets"
+	cmdutil "github.com/openfga/openfga/pkg/cmd/util"
+	"github.com/pressly/goose/v3"
+	"github.com/spf13/cobra"
+)
+
+const (
+	datastoreEngineFlag = "datastore-engine"
+	datastoreURIFlag    = "datastore-uri"
+	versionFlag         = "version"
+)
+
+func NewMigrateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "migrate",
+		Short: "Run database schema migrations needed for the OpenFGA server",
+		Long:  `The migrate command is used to migrate the database schema needed for OpenFGA.`,
+		RunE:  runMigration,
+	}
+
+	cmd.Flags().String(datastoreEngineFlag, "", "(required) the database engine to run the migrations for")
+	cmdutil.MustMarkFlagRequired(cmd, datastoreEngineFlag)
+
+	cmd.Flags().String(datastoreURIFlag, "", "(required) the connection uri of the database to run the migrations against (e.g. 'postgres://postgres:password@localhost:5432/postgres')")
+	cmdutil.MustMarkFlagRequired(cmd, datastoreURIFlag)
+
+	cmd.Flags().Uint(versionFlag, 0, `the version to migrate to. If omitted, the latest version of the schema will be used`)
+
+	return cmd
+}
+
+func runMigration(cmd *cobra.Command, _ []string) error {
+	engine, err := cmd.Flags().GetString(datastoreEngineFlag)
+	if err != nil {
+		return err
+	}
+
+	uri, err := cmd.Flags().GetString(datastoreURIFlag)
+	if err != nil {
+		return err
+	}
+
+	version, err := cmd.Flags().GetUint(versionFlag)
+	if err != nil {
+		return err
+	}
+
```

Comment:
```suggestion
goose.SetLogger(goose.NopLogger())
```
can we set goose to use the NopLogger so we don't get the log output from the applied migrations? If everything goes OK then we shouldn't get any log output IMO.

Reply:
Added. I kind of like the logs to get some feedback, but I can see your point too. Thanks!